forms-flow-web/src/components/CustomComponents/FormInput.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Form, InputGroup } from 'react-bootstrap';
+import { Translation } from "react-i18next";
+
+const FormInput = ({
+  type = "text",
+  label,
+  value = "",
+  onChange,
+  onBlur,
+  placeholder = "",
+  isInvalid = false,
+  feedback = "",
+  disabled = false,
+  size,
+  dataTestid,
+  ariaLabel,
+  className ,
+  required = false,
+  icon,
+  id
+}) => {
+
+  const inputClassNames = "Form-control-input " + className ;
```

Comment:
can we set className alone here instead of "Form-control-input " + className ? 
why we need it? @fahad-aot

Reply:
className for Additional custom classes and I have added some styles for the input in MF.